services/system/AccountSys/ui/src/hooks/useAccountsWithKeys.ts

```diff
@@ -37,7 +37,7 @@ export const useAccountsWithKeys = (): [AccountWithAuth[], (key: string) => void
             return uniqueAccounts([...currentAccounts, ...userAccounts])
         }));
 
-        Promise.all(keyPairs.map(keyPair => fetchAccountsByKey(keyPair.publicKey))).then(responses => {
+        Promise.all(keyPairs.filter(keyPair => keyPair.publicKey).map(keyPair => fetchAccountsByKey(keyPair.publicKey))).then(responses => {
```

Comment:
I don't think your Prettier is configured the same as ours. Long lines like this would be on a few lines.

Reply:
I've disabled my local one and selected the my docker container one to be the sole one instead, I'll take better notice of formatting happening assuming it's my formatter's fault than others.